app/src/main/java/com/example/poste/api/poste/models/User.java

```diff
@@ -7,18 +7,24 @@
 import com.example.poste.api.poste.exceptions.MalformedResponseException;
 import com.example.poste.api.poste.exceptions.NoUserFoundException;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class User {
 
     private int id;
     private String email;
     private String username;
     private String password;
+    private List<Folder> folders;
 
     public User(int id, String email, String username, String password) {
         this.id = id;
         this.email = email;
         this.username = username;
         this.password = password;
+
+        folders = new ArrayList<>();
```

Comment:
This doesn't hurt anything, but folders is instantiated during the User constructor

Reply:
Same issue as before, this is the old User.java file. it is soon to be incinerated.

Reply:
Same issue as before, this is the old User.java file. it is soon to be incinerated.

Reply:
Roger. Are we still making changes to these old models?

Reply:
no, not intentionally.
I will extract them while building the MVP so they don't get accidently used again